addons/sys_core/fnc_getGear.sqf

```diff
@@ -21,7 +21,9 @@ if (isNull _unit) exitWith {[]};
 
 // diag_log text format["Assigned Items: %1", (assignedItems _unit)];
 
-private _gear = (weapons _unit) + (items _unit) + (assignedItems _unit);
+private _gear = weapons _unit;
+_gear append (items _unit);
+_gear append (assignedItems _unit);
```

Comment:
We only ever check for `"ItemRadio"` and `"ItemRadioAcreFlagged"` from assigned items (always 7 long), which both fit only into radio slot.

This could be improved so:
```sqf
_gear pushBack ((assignedItems _unit) select 3);
```
And we get rid of looping through 6 items pointlessly.